interfaces/builtin/camera.go

```diff
@@ -33,6 +33,10 @@ const cameraConnectedPlugAppArmor = `
 # Until we have proper device assignment, allow access to all cameras
 /dev/video[0-9]* rw,
 
+# Access to the media controller API, to control V4L device connections
+# See https://www.kernel.org/doc/Documentation/userspace-api/media/mediactl/media-controller-intro.rst
+/dev/media[0-9]* rw,
```

Comment:
hmm, reading about the media-controller, this seems to provide not just reading / writing video frames but also _control_ of the hardware related to video subsystems such as adjusting the image produced like color correction, etc. which afaik the `/dev/video[0-9]*` accesses don't provide for. probably it's fine to leave it here in camera, but maybe cause for a `video-control` interface or something like that instead if folks think that kind of action should be more privileged than camera

Reply:
I like the idea of a new interface - I have noticed that we have also `/deb/v4l-subdev*` devices (https://www.kernel.org/doc/html/v4.12/media/uapi/v4l/dev-subdev.html), plus access to `/sys/` is needed for media-ctl cli. So I think I will drop this bit and maybe create another MP for a new interface if folks like this.

Reply:
yes, it seems a media-control interface would be appropriate for this part

Reply:
Added media-control interface to the PR